debezium-core/src/main/java/io/debezium/openlineage/dataset/DefaultOutputDatasetNamespaceResolver.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.openlineage.dataset;
+
+import java.util.function.Function;
+
+import io.debezium.config.Configuration;
+
+public class DefaultOutputDatasetNamespaceResolver implements OutputDatasetNamespaceResolver {
+
+    private static final String LIST_SEPARATOR = ",";
+    private static final String KAFKA_NAMESPACE_FORMAT = "kafka://%s";
+    private static final String SCHEMA_HISTORY_INTERNAL_KAFKA_BOOTSTRAP_SERVERS_PROPERTY = "schema.history.internal.kafka.bootstrap.servers";
```

Comment:
@jpechane I don't recall if this is in some way required for KC connectors. Can you confirm?

Reply:
So for the KC deployment this is the only option.

Reply:
Usually yes